docs/documentation/pinning.md

```diff
@@ -5,14 +5,141 @@ slug: /pinning
 sidebar_label: Pinning
 ---
 
-## 🚧 Under Construction 🚧
-:::caution 🚧 This page is under construction
+Pinning allows you to guarantee that your uploaded content will always be available by storing it **locally on your own Bee node**. However, pinning **guarantee availability across the Swarm network**. Therefore you must use pinning along with the **stewardship utilities** included in `bee-js` to monitor the availability of your pinned content and reupload it if needed.
```

Comment:
> Pinning allows you to guarantee that your uploaded content will always be available by storing it **locally on your own Bee node**.

Not only _your uploaded content_, any content, not required to be uploaded by you.

> However, pinning **guarantee availability across the Swarm network**.

Missing "DOES NOT"